cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php

```diff
@@ -52,7 +52,7 @@ class Video {
 	 *
 	 * @var string
 	 */
-	const PLAYER_VER = '1.3.3';
+	const PLAYER_VER = '1.4.0';
```

Comment:
@dugajean 

Can we test upgrading also the CORE_VER to `2.6.3` — https://github.com/cloudinary/pkg-cloudinary-core?